test/fixtures/code_samples/main.rb

```diff
@@ -0,0 +1,28 @@
+# Given an array of 0s and 1s in random order.
+# Segregate 0s on left side and 1s on right side of the array
```

Comment:
Could you convert this to a block comment ?
```suggestion
=begin
Given an array of 0s and 1s in random order.
Segregate 0s on left side and 1s on right side of the array
=end
```